app/assets/javascripts/snowcrash/pages/nodes/new_form.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  var copyOver = function($to, from, $typeTo, $typeFrom) {
```

Comment:
We're passing the `$to` element but not the `from` element. I'm assuming this is because we don't need the `from` element, only the value. But why are we passing the `$typeFrom` element and not the value?

Reply:
That's a good inconsistency catch. I'm going to make this change for `$typeFrom` to just be a passed value.